.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: UnitTests
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.dependency }} + Symfony ${{ matrix.symfony }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: true
+            matrix:
+                include:
+                    -   php: '5.5'
+                        dependency: lowest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '5.5'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '5.6'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '7.0'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '7.1'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: '^3.0'
+                    -   php: '7.2'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^4.0'
+                    -   php: '7.3'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^5.0'
+                    -   php: '7.4'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^5.0'
+
+        steps:
+            -   name: Checkout source
+                uses: actions/checkout@v2
+
+            -   name: PHP setup
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: ${{ matrix.coverage }}
+
+            -   name: Require Symfony version
+                if: matrix.symfony != 'any'
+                run: |
+                    composer global require --no-interaction --no-progress symfony/flex:^1.11
+                    composer config extra.symfony.require ${{ matrix.symfony }}
+
+            -   name: Update composer dependencies
+                uses: ramsey/composer-install@v1
+                with:
+                    dependency-versions: ${{ matrix.dependency }}
+
+            -   name: Run test suite
+                run: ./vendor/bin/phpunit -v
+
```

Comment:
you may add https://github.com/php-coveralls/php-coveralls/blob/master/.travis.yml#L44 already - let's build the phar file (even if it's not uploaded) on the PR, it will at least prove that file is build-able

Reply:
Done